profiles/cloudways.json

```diff
@@ -1,129 +1,311 @@
 {
   "name": "Cloudways",
-  "revision": "2014-05-28",
+  "revision": "2016-04-20",
   "url": "http://www.cloudways.com/en/",
   "status": "production",
   "status_since": "2012-03-21",
-  "type": "SaaS-centric",
+  "type": "PaaS-centric",
```

Comment:
There is no PaaS-centric tpye. Please see https://github.com/stefan-kolb/paas-profiles#type for more information. In your case SaaS-centric should be alright.